bin/version

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env bash
+
+tag=$1
+
+## If no tag is provided, default to latest
+if [ -z "$tag" ]; then
+  tag="latest"
+fi
+
+if [ "$tag" != "latest" ] && [ "$tag" != "nightly" ] && [ "$tag" != "experimental" ]; then
+  echo "Error: tag must be one of: latest, nightly, experimental"
+  exit 1
+fi
+
+if [ "$tag" = "latest" ]; then
+  pnpm changeset version
```

Comment:
Couldn't we include this in the `bin/release` script and get rid of this file?

Reply:
What about a no-op command? 
```
  version: echo "Versions are updated in the release script"
```

Just trying to simplify a bit, but we can try to improve it later.